README.md

```
@@ -1,208 +1,177 @@
 
-### 변수 및 함수
+## 서비스 사진 📸
 
-- 카멜 케이스 (`camelCase`)
-- 일반 함수: 역할이 명확한 이름
-- 이벤트 핸들러: `handle[Target][Action]` 형태
-- prop 콜백 함수: `on[Target][Action]` 형태
+<table>
+  <tr>
+    <td align="center">
+      <img src="https://github.com/user-attachments/assets/77f33a56-3960-49ea-8a7e-3ce8b0b1750c" width="250"/><br/>
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)